### DISCUSSION TITLE
VS Code Extension para agrupar arquivos de Razor Pages

### DISCUSSION THREAD

**Discussion by evertonfoz** [balta-io/forum#286] [Tira Dúvidas] [answered]
Olá pessoal. 
Alguém conhece e recomenda alguma extensão para usar no VS Code que agrupe os arquivos de uma página Razor?
Obrigado

**Comment by andrebaltieri** [comment, marked as answer]
@evertonfoz bom dia! O próprio VS Code tem essa funcionalidade:
https://dev.to/this-is-learning/file-nesting-in-vs-code-46l5
